Doc/library/os.path.rst

```diff
@@ -356,14 +359,28 @@ the :mod:`glob` module.)
    concatenation of *path* and all members of *\*paths*, with exactly one
    directory separator following each non-empty part, except the last. That is,
    the result will only end in a separator if the last part is either empty or
-   ends in a separator. If a segment is an absolute path (which on Windows
-   requires both a drive and a root), then all previous segments are ignored and
-   joining continues from the absolute path segment.
+   ends in a separator.
+
+   If a segment is an absolute path (which on Windows requires both a drive and
+   a root), then all previous segments are ignored and joining continues from the
+   absolute path segment. For example::
```

Comment:
Just to confirm, this example is non-Windows only right? namely macOS and Linux right? If so, I would add "On Linux, for example::" (the paragraph is not exactly Linux specific but the example is)